### PR TITLE
Update qiao2019micro-batch_2.md

### PR DESCRIPTION
**Pull request by nilesh2797**

### CODE REVIEW

File: _summaries/qiao2019micro-batch_2.md [hunk 1]
```diff
@@ -1,19 +1,60 @@
...
+<img src="qiao2019micro-batch_2/bn-singularity.png" width="500"/> | <img src="qiao2019micro-batch_2/ws-bn.png" width="500"/>
+
+## Batch-Channel Normalization (`BCN`)
+Apply both batch and channel normalization together to activations
+* For Large Batches: `BCN`(X) = `CN`(`BN`(X))
+* For Micro Batches: Keep running estimates of mean and variance for `BN`
+
+> Channel based normalizations make estimates based normalizations possible, and estimate based normalization helps channel based normalizations avoid elimination singularities
+
+**TODO**: Add results
```
Comment (jordi1215) [inline _summaries/qiao2019micro-batch_2.md:56]: Don't forget to add your results if you are still planning to

File: _summaries/qiao2019micro-batch_2/dummy.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (jordi1215) [inline _summaries/qiao2019micro-batch_2/dummy.txt:1]: What is this file for???

File: _summaries/qiao2019micro-batch_2.md [hunk 3]
```diff
@@ -1,19 +1,60 @@
...
 
 ## TL;DR
-* Three
-* Bullets
-* To highlight the core concepts
+* Weight Standardization can give similar effects of Batch Normalization even with very small batch sizes 
+* With smaller batch sizes, Batch normalization when applied with Channel normalization improves the training
```
Comment (jordi1215) [inline _summaries/qiao2019micro-batch_2.md:60]: Great paper! It looks super good! Very concise and straight to the point. I have nothing else to add. This summary helped me understand the original paper for sure. Good job!

**Review by jordi1215 — APPROVED** [review]

File: _summaries/qiao2019micro-batch_2.md (hunk 1 above)
Comment (nilesh2797) [reply _summaries/qiao2019micro-batch_2.md]: Done.

File: _summaries/qiao2019micro-batch_2/dummy.txt (hunk 2 above)
Comment (nilesh2797) [reply _summaries/qiao2019micro-batch_2/dummy.txt:1]: Github doesn't allow to create empty directory so added this dummy file

File: _summaries/qiao2019micro-batch_2.md (hunk 3 above)
Comment (nilesh2797) [reply _summaries/qiao2019micro-batch_2.md:63]: Thanks!
